server.ts

```diff
@@ -0,0 +1,58 @@
+import bodyParser from "body-parser";
+import express from "express";
+// QUESTION - diff between above statement?
```

Comment:
In general:

no curly brackets on import -> imports the default export from that file...you get to decide how it's named
curly brackets on import -> imports a specific function/variable/object from that file...that source file defines how it's named